config/locales/candidate_interface/personal_statement.yml

```diff
@@ -4,10 +4,16 @@ en:
       becoming_a_teacher:
         change_action: why do you want to be a teacher?
         label: Why do you want to be a teacher?
+        complete_hint_text_html: |
+          Proofread this section carefully for spelling and grammar. Ask someone you trust for their opinion.
+          <p>Ensure this is all your own work as plagiarism will be penalised.</p>
       subject_knowledge:
         key: Your knowledge about the subject you want to teach
         change_action: evidence of subject knowledge
         label: Tell us what you know about the subject you want to teach
+        complete_hint_text_html: |
+          Proofread this section carefully for spelling and grammar. Ask someone you trust for their opinion.
```

Comment:
```suggestion
          <p>Proofread this section carefully for spelling and grammar. Ask someone you trust for their opinion.</p>
```

Reply:
Should we also add the `govuk-body` class to this paragraph?

Reply:
No, because this text appears within <div class="govuk-hint">…</div>. Not sure I like that much, either, but not sure how we could improve?

Reply:
Could potentially pass in an array of hints (paragraphs) into the component and loop through, if that's preferred

Reply:
That would work! That way you could have the following:

```
<p class="govuk-hint">One line hint</p>
```

or 

```
<p class="govuk-hint">Two line hint, first line</p>
<p class="govuk-hint">Two line hint, second line</p>
```